docker-compose.yml

```diff
@@ -10,10 +10,16 @@ services:
       - GAS_LIMIT=6700000
       - IPFS_HOST=ipfs
       - IPFS_PORT=5001
-      - IPNS_PATH=/ipns/
+      - ESCROW_BUCKETNAME=escrow-results
+      - ESCROW_AWS_ACCESS_KEY_ID=minio
+      - ESCROW_AWS_SECRET_ACCESS_KEY=minio123
+      - ESCROW_ENDPOINT_URL=http://minio:9000
```

Comment:
how do we control what aws region to hit?

Reply:
good point, adding